### DISCUSSION TITLE
[Problem] Unocss && writeToDisk infinite hmr  loop when dev

### DISCUSSION THREAD

**Discussion by zhtyytg** [web-infra-dev/rsbuild#1778] [Q&A] [answered]
related to this issue  https://github.com/web-infra-dev/rsbuild/issues/654#issue-2007373171
When I use `unocss/postcss` and `config writeToDisk:true` at rsbuild.config.ts. The hmr will infinitely loop after change any component code.
And I found that the issue was fixed by change writeToDisk default to false.
So it means that using both of them are not allowd in rsbuild?

**Comment by 9aoy** [comment, marked as answer]
maybe you should update your unocss config like this:
```
export default defineConfig({
  content: {
    filesystem: [
      'src/**/*.{html,js,ts,jsx,tsx,vue,svelte,astro}',
    ]
  },
})
```
https://github.com/web-infra-dev/rsbuild/issues/654#issuecomment-1823786489

**Reply by zhtyytg** [reply]
It helps, ty!
